wagtail/admin/templates/wagtailadmin/pages/listing/_locked_indicator.html

```diff
@@ -7,6 +7,11 @@
 {% endcomment %}
```

Comment:
It would be good to add a line to the comment section here relating to this new behaviour.